### DISCUSSION TITLE
Feedback after detailed testing of the Anaconda installer

### DISCUSSION THREAD

**Discussion by 4sokol** [rhinstaller/anaconda-webui#33] [General]
**Infrastructure for testing**: KVM/QEMU (qemu-6.2.0-16.fc36).

**Comments**:
1. The navigation buttons look unfamiliar, unaccustomed location + unaccustomed order (a more familiar view is the location of the buttons on the right side, the order of the buttons: cancel - back - forward)
![1 navigation buttons](https://user-images.githubusercontent.com/72856789/205460202-305da32e-8ca6-4bb3-81cb-640653758bb4.png)
2.  #BUG. When selecting the Czech language, part of the interface is not translated (on the screenshot, highlighted in red) 
![2 translation](https://user-images.githubusercontent.com/72856789/205460274-0250d4d9-11cb-405a-bff6-6cc52298c4a6.png)
![2 1 translation](https://user-images.githubusercontent.com/72856789/205460291-7c3d541e-e7e9-488d-8c68-9c9aea207ee3.png)
![2 2 translation](https://user-images.githubusercontent.com/72856789/205460315-c57c03ba-0932-425f-8981-15a34ffd65d7.png)
3. Not implemented hard disk partitioning: selection of file system, partitions, etc.
4. Keyboard layout selection is not implemented
5. Not implemented the ability to select desktop environment
6. Not implemented the ability to configure the network card, Wi-Fi, proxy, etc.
7. The ability to configure users is not implemented
8. After a complete installation of the OS and rebooting of the system, a setup window appears, when entering the user's login and password, there is no possibility to add admin rights (wheels)
9. I attach a screenshot of the drive partitions, which were set by default, without selecting the changes during the installation (drive 'vda' 40 GiB)
![3 partitions](https://user-images.githubusercontent.com/72856789/205460409-0f08db69-c7ed-4cf9-a531-c9b3bae1c5d9.png)
10. By default, the system assigns "#10 (wheels)" permissions to the new user
11. The keyboard layout is English after first OS run, even when choosing a different system language during OS installation (I chose Czech for testing)

**Short Conclusion:** when testing this installer, no major bugs were detected, there were problems with the language interface during the entire installation, the main drawback I think is very limited settings before installing the Operating System itself, this installer has a very limited

**Comment by nikodunk** [comment]
Agree with point 1 - the order of the buttons would be more intuitive and standard if reversed and right aligned. Thanks for your hard work on this!!

**Comment by vzsh** [comment]
Is there any plan to develop the custom partition function? I really hope to see this function in the next version.

**Comment by KKoukiou** [comment]
> **Infrastructure for testing**: KVM/QEMU (qemu-6.2.0-16.fc36).
> 
> **Comments**:
> 
>     1. The navigation buttons look unfamiliar, unaccustomed location + unaccustomed order (a more familiar view is the location of the buttons on the right side, the order of the buttons: cancel - back - forward)

This has been fixed

> 
>     2. #BUG. When selecting the Czech language, part of the interface is not translated (on the screenshot, highlighted in red)

Getting the translations for newly introduced strings is expected to take some time. It's impossible to get high translation percentage immediately for all languages.

>     3. Not implemented hard disk partitioning: selection of file system, partitions, etc.

This should be done through blivet-gui (soon to be cockpit-storage). These apps are available by the newly 'Modify storage' button.

> 
>     4. Keyboard layout selection is not implemented

Will not be implemented for the time being.

> 
>     5. Not implemented the ability to select desktop environment

Web UI is only available for now in Live Workstation images we static payload. This is expected to not be available there.

> 
>     6. Not implemented the ability to configure the network card, Wi-Fi, proxy, etc.

For the Live Workstation ISO (our only supported target deliverable), users are expected to do that from the gnome environment. 

> 
>     7. The ability to configure users is not implemented

Was introduced last week: https://github.com/rhinstaller/anaconda/pull/5280

> 
>     8. After a complete installation of the OS and rebooting of the system, a setup window appears, when entering the user's login and password, there is no possibility to add admin rights (wheels)

Was fixed with: https://github.com/rhinstaller/anaconda/pull/5280
> 
>     9. I attach a screenshot of the drive partitions, which were set by default, without selecting the changes during the installation (drive 'vda' 40 GiB)

> **Short Conclusion:** when testing this installer, no major bugs were detected, there were problems with the language interface during the entire installation, the main drawback I think is very limited settings before installing the Operating System itself, this installer has a very limited

Thanks for testing, it's almost one year later and many improvement were introduced. It would be great if you can give a follow up feedback!
